Tasks/AzureFileCopyV5/Tests/L0AddAzureVMCustomScriptExtension.ps1

```diff
@@ -0,0 +1,68 @@
+[CmdletBinding()]
+param()
+
+. $PSScriptRoot\..\..\..\Tests\lib\Initialize-Test.ps1
+. $PSScriptRoot\MockVariable.ps1
+. $PSScriptRoot\..\Utility.ps1
+. $PSScriptRoot\MockHelper.ps1
```

Comment:
have you checked whether this newly written tests are running along with all other tests from CI/CD pipeline which runs on PR